src/libs/Navigation/types.ts

```diff
@@ -854,7 +854,7 @@ type SettingsNavigatorParamList = {
     };
     [SCREENS.WORKSPACE.COMPANY_CARDS_SETTINGS_FEED_NAME]: {
         policyID: string;
-        currentFeedName?: string;
+        feedName?: string;
```

Comment:
```suggestion
        selectedFeed: string;
```

Reply:
Please update in other places

Reply:
@DylanDylann We can't use `selectedFeed` because it will make things more complex. If you want to use `selectedFeed`, we must also send the `cardFeeds` object in the parameters.

Reply:
If we use selectedFeed without passing the cardFeeds object, we encounter an issue like the video below because cardFeeds is retrieved from Onyx.
<details>
<summary>VIDEO</summary>
https://github.com/user-attachments/assets/ee547f38-cdd2-4de7-8517-11ee8f394c4b
</details>

Reply:
In src/pages/workspace/companyCards/WorkspaceCompanyCardsSettingsFeedNamePage.tsx, we can get cardFeed from Onyx, right?

Reply:
yes

Reply:
Could you elaborate in more detail on what problem you want to mention [here](https://github.com/Expensify/App/pull/52301#discussion_r1836132308)?

Reply:
I'm still not clear on your comment. Can you explain in more detail?

- [Here](https://github.com/Expensify/App/pull/52301#discussion_r1836102753), We're just renaming that, right?
- [Here](https://github.com/Expensify/App/pull/52301#discussion_r1836101691), Do you want me to send selectedFeed instead of feedName?

In my opinion, we can't use selectedFeed to set the parameters because we have a bug when using it, and I mentioned the reason in the comment. If we rename it, then I think we should rename feedName to selectedFeedName

Reply:
Let's pass selectedFeed in the param and then we can handle your bug in another way. Please check [this branch](https://github.com/DylanDylann/App/tree/setting-feed-name-new-params) if there is any confusion. Thanks.

Reply:
@DylanDylann I understand your point; you're suggesting we should use selectedFeed instead of selectedFeedName. However, if we use selectedFeed, we encounter the issue I showed in the video. Are you aware of this, and if so, do you have any suggestions for handling it? Because if we get the value from [cardFeeds](https://github.com/DylanDylann/App/blob/e61493bb25f87d71fb0a83f1b36a5d47392f25a4/src/pages/workspace/companyCards/WorkspaceCompanyCardsSettingsFeedNamePage.tsx#L45), we won’t need to set additional parameters.

Reply:
small note: Never give a video without any description and say it is a bug like in the video. When elaborating on a bug please say detailed steps, actual behavior, and expected behavior or at least a description to other people can understand that you want to say

> we encounter an issue like the video below because cardFeeds is retrieved from Onyx.

Reply:
>  Are you aware of this, and if so, do you have any suggestions for handling it?

It seems I handled it in my ref branch

Reply:
> we won’t need to set additional parameters.

Also fine, if that we need to handle loading indicator if lastSelectedFeed isn't loaded

Reply:
@DylanDylann , Can you test this case on your branch?
https://github.com/user-attachments/assets/ed1f1681-6527-469d-b97b-cf07a5eb8cea

Reply:
> Also fine, if that we need to handle loading indicator if lastSelectedFeed isn't loaded

@DylanDylann I will update it to use selectedFeed to set the parameters and add a loading indicator to wait for the [cardFeeds](https://github.com/Expensify/App/blob/ae40ea5843e18a05a42222fe78a29aacf06c5b46/src/pages/workspace/companyCards/WorkspaceCompanyCardsSettingsFeedNamePage.tsx#L42) value. Do you agree?

Reply:
Got it. Will update. thank you

Reply:
> Can you test this case on your branch?

@huult It worked. Is there any problem?

https://github.com/user-attachments/assets/93387765-15bd-4ba5-b10f-9c532a7bac23